common/welcome.inc

```diff
@@ -53,6 +53,8 @@ Sub we.run_menu()
 End Sub
 
 Sub we.run_program(f$, cmdline$)
+  CLS
```

Comment:
Prefer `Cls` in my code.

Should we also do:
`Mode 1,8`
`Page 0`
before the `Cls` ?

Reply:
Probably we should do that. Otherwise the code assumes it was in a suitable mode and Page 0 was already either set by the menu, or by we.end_program().

Making suggested changes (but Page Write 0).